libraries/AP_TemperatureSensor/TSYS01.cpp

```diff
@@ -1,8 +1,8 @@
 #include "TSYS01.h"
 
-#include <utility>
 #include <stdio.h>
-#include <AP_HAL/AP_HAL.h>
+#include <utility>
+#include <AP_HAL/Semaphores.h>
```

Comment:
Why is this required in the cpp file?

Reply:
yep, in fact it is not even needed ...my bad